GOVERNANCE.md

```diff
@@ -1,137 +1,79 @@
-several timezones, the TSC will seek to resolve as many agenda items as possible
-outside of meetings using
-[the TSC issue tracker](https://github.com/nodejs/TSC/issues). The process in
-the issue tracker is:
-
-* A TSC member opens an issue explaining the proposal/issue and @-mentions
-  @nodejs/tsc.
-* After 72 hours, if there are two or more `LGTM`s from other TSC members and no
-  explicit opposition from other TSC members, then the proposal is approved.
-* If there are any TSC members objecting, then a conversation ensues until
-  either the proposal is dropped or the objecting members are persuaded. If
-  there is an extended impasse, a motion for a vote may be made.
+Teams exist in order to steward certain parts of the project.
+
+All team members are elected by method of volunteering. Any contributor to a
+team's efforts can volunteer for said team, or be nominated by a team member.
+All nominations are publicly reviewed and voted on by the existing team members.
+
+If a team member decides to step down, they can do so instantly. If a team
+member is found to be violating the project's Code of Conduct, they will be
+moderated and potentially permanently ejected from the team.
+
+There should always be at least 3 members of any team.
+
+There are a number of fixed teams:
+
+#### Core team
+
+The core team is a team with the purpose of deciding the general direction of
+the project and managing cross-cutting concerns. They only decide over the most
+high-level aspects, all other matters should be delegated to another, more
+specific team.
+
+Every member of the core team is also an Owner of the [ayojs]() GitHub
+organization.
+
+#### Moderation team
+
+The moderation team serves to enforce the project's Code of Conduct and manage
+situations of conflict. There are no core team members on the moderation team,
+in order to not pick sides.
+
+Every member of the moderation team is also an Owner of the [ayojs]() GitHub
+organization.
+
+#### Sub-teams
+
+Other teams focus on one particular area of the project. They have full autonomy
+over said area, their decisions do not need to be approved by any core team
+member. Every sub-team needs to have one member that is also a member of the
+core team, in order to notice and manage cross-cutting concerns.
+
+Each team decides who gets permissions for their respective
+repository/repositories.
+
+More information about these teams can be found on the [website]().
+
+_TODO: Add website link?_
+
+## Elected Final Arbiters
+
+Elected Final Arbiters (EFA) are Members of the Ayo.js project that serve to
+resolve issues where there is no consensus. They are elected on a per-issue
+basis.
+
+There should always be multiple EFAs, a recommended number is 3-5.
+
+EFAs have the authority to establish a decisive vote on issues where no
+consensus can be reached by the team(s). They do not place
+above the teams in the hierarchy, for they are Members. A EFA should not be
```

Comment:
"An" EFA?

Reply:
Also I'm interested how an EFA cannot be a member of the Core team. With our current size, this seems hard to do.

Reply:
Yeah, I agree with @varjmes here, I'd say the only restriction for the EFA(s) is that they haven't already been apart of the issue in question, which will be hard enough to find early on without imposing any further restriction.